decksite/achievements.py

```diff
@@ -94,7 +94,6 @@ def display(self, p: 'person.Person') -> str:  # pylint: disable=no-self-use, un
     def load_summary(self, season_id: Optional[int] = None) -> Optional[str]:
         season_condition = f'AND season_id = {season_id}' if season_id != 'all' else ''
         sql = f"""SELECT SUM(`{self.key}`) AS num, COUNT(DISTINCT person_id) AS pnum FROM _achievements WHERE `{self.key}` > 0 {season_condition}"""
-        print(sql)
```

Comment:
@adambiltcliffe Please don't use print() on decksite.  It's really bad for performance.